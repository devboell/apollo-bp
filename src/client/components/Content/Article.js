import styled from 'styled-components'

const Article = styled.article`
  display: flex;
  justify-content: flex-start;
  margin: 20px 50px;
  flex-basis: 80%;
`

export default Article
