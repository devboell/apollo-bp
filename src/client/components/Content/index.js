import React from 'react'
import { Route } from 'react-router-dom'
import Introduction from 'components/Introduction'
import Gallery from 'components/Gallery'
import Article from './Article'

const Content = () =>
  <Article>
    <Route exact path="/" component={Introduction} />
    <Route path="/:artistName" component={Gallery} />
  </Article>

export default Content
