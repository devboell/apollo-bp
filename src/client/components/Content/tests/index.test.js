import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import Content from '../index'

it('List component snapshot', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Content />
    </MemoryRouter>,
  ).toJSON()
  expect(tree).toMatchSnapshot()
});
