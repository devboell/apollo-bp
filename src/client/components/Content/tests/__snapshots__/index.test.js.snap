// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component snapshot 1`] = `
<styled.article>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/:artistName"
  />
</styled.article>
`;
